### DISCUSSION TITLE
Ebo added as integration/add on

### DISCUSSION THREAD

**Discussion by Mstrrlm** [home-assistant/frontend#17525] [Dashboard]
This’s a surveillance type robot I would like to see this asap and to be able to set an automation to patrol an area the app doesn’t do it for a max of 3 minutes similar to a robot vacuum.  It’s best if I just provide the URL. 
https://www.enabot.com/collections/ebo-air

Hope I did this correctly and thanks

**Comment by frenck** [comment]
Thank you for reaching out. This is not a frontend related request, but one for a new integration.

You should try our [Community Forum: Feature Requests](https://community.home-assistant.io/c/feature-requests) forum instead.

If you have additional questions, feel free to join our [Discord chat server](https://discord.gg/c5DvZ4e).

Thanks! 👍
